pkg/providers/vsphere/virtualmachine/backup.go

```diff
@@ -301,9 +302,19 @@ func getDesiredVMResourceYAMLForBackup(
 func trimBackupFields(obj client.Object) {
 	if annotations := obj.GetAnnotations(); len(annotations) > 0 {
 		delete(annotations, corev1.LastAppliedConfigAnnotation)
+
 		obj.SetAnnotations(annotations)
```

Comment:
I think there are additional values that we'd want to remove too - https://github.com/vmware-tanzu/vm-operator/blob/4ac019e5dcfe0fbb27a5ebfe372378355e3133d5/pkg/constants/constants.go#L127-L147. Thoughts?

Reply:
Absolutely agree.  However, I want to club those in a different PR because I want one PR with all the labels and annotation and for that, I want to get an ACK from VCFA.

Is it okay if I keep this one scoped to the DR operator ask?

Reply:
Sure thing